packages/gatsby-theme-docs/src/layouts/internals/layout-page-with-hero.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { ThemeProvider } from 'emotion-theming';
+import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
+
+// TODO: for now we assume that the website provides a file `homepage-hero.svg`
+// in the `static` folder.
+const homepageHeroSvgUri = '/homepage-hero.svg';
+
+// The container is used to render a background image with for the full page width.
+const Container = styled.div`
+  grid-area: page-header;
+  position: relative;
+  background-color: ${(props) => props.theme.colors.light.primary};
+  background-image: url('${homepageHeroSvgUri}');
+  background-attachment: fixed;
+  background-size: cover;
+
+  @media screen and (${designSystem.dimensions.viewports.tablet}) {
+    background-size: auto;
+  }
+  @media screen and (${designSystem.dimensions.viewports.largeDesktop}) {
+    background-size: cover;
+  }
+
+  /* License with attribution of the background by SVGBackgrounds.com */
+`;
+// This wrapper element increases the padding for the "normal" content page.
+const ContentWrapper = styled.div`
+  padding: 0 ${designSystem.dimensions.spacings.s};
+
+  @media screen and (${designSystem.dimensions.viewports.tablet}) {
+    padding: 0 ${designSystem.dimensions.spacings.xxl};
+  }
+`;
+// Constraint the heading to the "page content" width.
+const Title = styled(Markdown.H1)`
+  padding: ${designSystem.dimensions.spacings.big}
+    ${designSystem.dimensions.spacings.m};
+  width: calc(100% - ${designSystem.dimensions.spacings.m} * 2);
+  max-width: ${designSystem.dimensions.widths.pageContent};
+
+  @media screen and (${designSystem.dimensions.viewports.tablet}) {
+    width: calc(100% - ${designSystem.dimensions.spacings.large} * 2);
+    padding: ${designSystem.dimensions.spacings.big}
+      ${designSystem.dimensions.spacings.large};
+  }
+`;
+
+const LayoutPageWithHero = (props) => (
+  <>
+    <Container heroBackgroundURL={props.heroBackgroundURL}>
+      <ThemeProvider
+        theme={{
+          colors: {
+            light: {
+              primary: designSystem.colors.light.textInverted,
```

Comment:
I assume that the heading color should be the "inverted" one, so basically the "white" color, given that the websites primary colors are darker colors.

This needs to be checked with @Shecki though. If necessary, the text color should also be configurable.

Reply:
In our conversation with him he said that the text will always be pure white when over the background image.

Reply:
Yes, the `inverted` color points to: `color-surface`

<img width="573" alt="image" src="https://user-images.githubusercontent.com/1110551/82025973-5c878b80-9692-11ea-83fa-637fa50a39a1.png">